modules/openy_demo_menu/modules/openy_demo_menu_main/config/install/migrate_plus.migration.openy_demo_extended_menu_link_main.yml

```diff
@@ -98,9 +98,6 @@ process:
     plugin: default_value
     source: language
     default_value: en
-  bundle:
```

Comment:
Should we bump the version to v2 of demo content? @mkdok @david-storm 

I believe with this change we'd break previous demo content of still available version of the distro

Reply:
@podarok seems yes, we should have new version of this module there

Reply:
done https://github.com/open-y-subprojects/openy_demo_content/tree/v2